src/plugins/sensors/consume.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from airflow.exceptions import AirflowException
+from airflow.operators.sensors import BaseSensorOperator
+from airflow.utils.decorators import apply_defaults
+
+from operators.base import BaseMessageManager
+
+log = logging.getLogger(__name__)
+
+
+class ConsumeMessageSensor(BaseSensorOperator, BaseMessageManager):
+    template_fields = tuple()
+    ui_color = '#B5F2FF'
+
```

Comment:
Probably assign #B5F2FF to a variable with a logical name (eg light_blue) and the assign this to u_color